kadasrouting/gui/navigationpanel.py

```diff
@@ -209,8 +209,8 @@ def hide(self):
         self.stopNavigation()
 
     def updateNavigationInfo(self, gpsinfo):
-        # prevent infinite loop for mocked gps
-        self.gpsConnection.statusChanged.disconnect(self.updateNavigationInfo)
+        # FIXME: prevent infinite loop for mocked gps
+        # self.gpsConnection.statusChanged.disconnect(self.updateNavigationInfo)
```

Comment:
@volaya  I am not sure here. Was it correct to comment out this line?

Reply:
Yes, it's correct. This line was needed for the "GPS" movement by panning the map.

Reply:
It will need to remove line 230 also since it's no longer needed.